packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VideoController/VideoController.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Button, Select, ThemeContext } from 'grommet'
+import { CirclePlay, PauseFill } from 'grommet-icons'
+import counterpart from 'counterpart'
+import en from './locales/en.json'
+import FormattedTime from './FormattedTime'
+import Slider from './Slider'
```

Comment:
Our convention is to put components into a `components` directory, so this would be `./components/FormattedTime` and `./components/Slider`.